app/konnect/dedicated-cloud-gateways.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Dedicated Cloud Gateways
+content_type: concept
+---
+
+Dedicated Cloud Gateways are API gateways that are fully managed by Kong in the cloud provider of your choice. 
+	
+With Dedicated Cloud Gateways, you don't need to host any pieces of the infrastructure yourself. You maintain control over the size and location of the gateway infrastructure, while Kong oversees the management of each instance and the entire cluster for you. This allows Kong to autoscale your nodes for you and reduces your operational complexity.
+
+Dedicated Cloud Gateways are the fastest way to configure and create a {{site.base_gateway}} in {{site.konnect_short_name}}. All you have to do is specify the security you want to use and pre-warm the cluster while {{site.konnect_short_name}} handles the cluster creation.
+
+Dedicated Cloud Gateways also have the following benefits:
+* SOC1 compliant out-of-the-box
+* {{site.konnect_short_name}} handles gateway upgrades for you
+* Supported on the following AWS regions: Sydney, Tokyo, Singapore, Frankfurt, Ireland, London, Ohio, Oregon
+* Choose public or private mode to decide who can view your APIs. In public mode, powered by Kong's public Edge DNS for clusters, you can expose your APIs to the internet. 
+
+You can manage your Dedicated Cloud Gateway nodes in [Gateway Manager](https://cloud.konghq.com/gateway-manager/).
+
+![cloud gateway dashboard](/assets/images/products/konnect/gateway-manager/konnect-control-plane-cloud-gateway.png)
+> _**Figure 1:** Example of the dedicated cloud gateway data plane node dashboard in Gateway Manager. The dashboard displays the requests, error rate, and P99 latency. It also displays the cluster configuration, as well as the data plane groups and regions._
+
+Dedicated Cloud Gateways are useful for the following use cases:
+
+| Use case | Cloud gateway solution |
```

Comment:
```suggestion
| Use case | Solution |
```

I'm not really a fan of cloud gateway solution. Problem: solution doesn't feel right.